src/components/Homepage/LendByCategory/LoanCategory.vue

```diff
@@ -72,12 +55,13 @@
 					</div>
 				</div>
 			</div>
-			<span
+			<button
 				class="arrow right-arrow"
 				:class="{inactive: scrollPos <= minLeftMargin}"
-				ref="rightArrow"
```

Comment:
Since there's no text inside this button we need some assistive text like `<span class="show-for-sr">Show more loans (?) </span>` inside the button